pl/main.lang

```diff
@@ -141,7 +141,7 @@ Firefox: nie dla zysku, niekorporacyjny, nieuległy. Wybierając Firefoksa wybie
 
 
 ;Donate
-Dotacje
+Darowizny
```

Comment:
I don't think this is correct, see also http://bugs.aviary.pl/show_bug.cgi?id=4981